src/drive/web/modules/views/OnlyOffice/helpers.js

```diff
@@ -14,6 +14,22 @@ export const isOfficeEnabled = () => {
 export const isOnlyOfficeModeReadOnly = ({ data }) =>
   data.attributes.onlyoffice.editor.mode === 'view'
 
+export function canWriteOfficeDocument() {
+  const office = flag('drive.office')
+  if (office) {
+    return office.write
+  }
+  return false
+}
+
+export function redirectToOnlyOfficePaywall(nextState, replace) {
+  if (!canWriteOfficeDocument()) {
```

Comment:
I don't have clear mind about the flags and how we will handle the production's deployment of this new Drive version. 

We'll need to add this `drive.office` flag to a few contexts thats all? 

Don't we need to change the stack code in order to be able to use the new `drive.office.write` flag to return the right editorMode value (used in useConfig)

Reply:
As I have elaborated, we will need the `drive.office` flag in all contexts after this PR as well as `drive.office.write` properly set. We could modify the stack so that it already returns `drive.office` so that the contexts in the users would already be correct before updating the version of Drive with this PR.